content/md/first-steps/manage-reference-entities.md

```diff
@@ -0,0 +1,233 @@
+---
+id: manage-reference-entities
+themes: catalog-settings, manage-your-reference-entities
+title: Manage your **reference entities**
+popular: false
+ee-only: true
+related: what-about-reference-entities, enrich-your-reference-entity-records
+---
+
+# Display the reference entities
+
+To display the existing [reference entities](/articles/what-about-reference-entities.html), click on `Entities` menu.   
+
+By default, no reference entity is created, from this single place, you can:
+- [Create your 1st reference entity  ](/articles/manage-reference-entities.html#create-a-reference-entity)
+- [Define its properties ](/articles/manage-reference-entities.html#define-its-properties) (`Properties` tab)
+- [Define its records attributes ](/articles/manage-reference-entities.html#define-its-records-attributes) (`Attributes` tab)
+- [Create and enrich its records ](/articles/enrich-your-reference-entity-records.html) (`Records` tab)
+- [Define its permissions](/articles/manage-reference-entities.html#set-permissions-on-a-reference-entity), which user groups can view or edit this reference entity and its records  (`Permissions` tab)
+
+If a reference entity has already been created, simply click on it to access its information.
+
+![image](../img/ReferenceEntity_ListEntities.png)
+
+# Create a reference entity
+
+To create a [reference entity](/articles/what-about-reference-entities.html), follow these steps:
+1.  Go to `Entities` menu
+1.  Click on `Create` button
```

Comment:
click on "the" create button?

Reply:
Yes you can add "the", but it is ok if you use " click on "Create" button ". It could be also: Click the "Create" button

Reply:
done